frontend/packages/core/src/button.tsx

```diff
@@ -146,22 +147,31 @@ const StyledIconButton = styled(MuiIconButton)<{ palette: ButtonPalette }>(
   {
     height: "48px",
     width: "48px",
```

Comment:
since we're adding two sizes now, can we also update the height & width for the small to be 32*32. seems like internally we also have done overrides to make the icons smaller. example: https://github.com/lyft/clutch/blob/main/frontend/workflows/projectSelector/src/card.tsx#L19-L21

and then we can use this small variant for the NPS wizard

Reply:
Updated